frontend/pages/admin/OrgSettingsPage/cards/Info/Info.tsx

```diff
@@ -124,6 +124,8 @@ const Info = ({
               onBlur={validateForm}
               error={formErrors.org_logo_url}
               inputWrapperClass={`${cardClass}__logo-field`}
+              tooltip="Avatar is displayed in the top bar and other areas of Fleet that
```

Comment:
Nit: Calling the org logo an "avatar" has always seemed a little weird to me. Could we just call this "logo" or "icon" instead? cc @noahtalerman